lambda/custom/constants.js

```diff
@@ -3,9 +3,11 @@
 module.exports = Object.freeze({
   // App-ID. TODO: set to your own Skill App ID from the developer portal.
   appId: '',
-
   dynamoDBTableName: 'ScoutDB',
-
+  TITLE_CHUNK_LEN: 5,
+  USER_AGENT: 'Alexa Skill',
```

Comment:
`USER_AGENT` is the only value from the previous {constants.strings} object that was being referenced from anywhere other than the unit tests. So I promoted it here to be a regular constant and removed it from the brand files.